packages/manager/modules/pci/src/projects/project/storages/object-storage/users/index.js

```diff
@@ -11,6 +11,8 @@ import component from './users.component';
 import routing from './users.routing';
 import deleteUser from './delete';
 import importPolicy from './import';
+import downloadRclone from './download-rclone';
+import service from '../../../users/users.service';
```

Comment:
**issue:** Cannot redefine the service which exists in other folders.

Reply:
Taken care